crates/aptos/e2e/cases/stake.py

```diff
@@ -78,9 +78,9 @@ def test_stake_add_stake(run_helper: RunHelper, test_name=None):
     )
 
     result = json.loads(response.stdout)["Result"]
-    if result[0].get("total_stake") != 2000000:  # initial 1M + added 1M
+    if result[0].get("total_stake") != 2:  # initial 1 + added 1
```

Comment:
```suggestion
    if result[0] == None or result[0].get("total_stake") != 2:  # initial 1 + added 1
```

Not sure if this really matters, but for consistency's sake, just noting this here

Reply:
Good catch!